tools/update-component-sha.sh

```diff
@@ -1,71 +1,5 @@
 #!/bin/sh
 set -e
 
-##
-#
-# script to replace hashes in config files
-# see usage() for usage and functionality
-#
+curl -L https://raw.githubusercontent.com/linuxkit/linuxkit/46ef83c97931753c27f1ee4f5616d437d0d8f2d6/scripts/update-component-sha.sh | sh -s - "$@"
```

Comment:
No, no -- that's not good. I really don't like this style of `curl | sh` at all (for obvious reasons).

Reply:
You are causing trouble! Well, as long as I am not the one who has to keep it up to date...